### PR TITLE
chore: release 0.9.8

### PR DESCRIPTION
**Pull request by moolen**
Bump helm chart for release 0.9.8.

diff: https://github.com/external-secrets/external-secrets/compare/v0.9.7...v0.9.8

e2e managed test runs:

https://github.com/external-secrets/external-secrets/actions/runs/6706003324/job/18221591798
https://github.com/external-secrets/external-secrets/actions/runs/6706004130/job/18221594285
https://github.com/external-secrets/external-secrets/actions/runs/6706001542/job/18221586045

### CODE REVIEW

File: .github/workflows/helm.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ jobs:
           python-version: 3.7
 
       - name: Set up chart-testing
-        uses: helm/chart-testing-action@v2.4.0
+        uses: helm/chart-testing-action@v2.3.1
```
Comment (moolen) [inline .github/workflows/helm.yml:39]: v2.4.0 is broken at the moment, see https://github.com/helm/chart-testing-action/issues/132
Downgrading fixes the issue. There's a fix on main but hasn't been released yet.

Comment (Skarlso) [reply .github/workflows/helm.yml:39]: Can you leave a comment for the downgrade with a reason? Something like:
```
# see https://github.com/helm/chart-testing-action/issues/132 before upgrading
uses: helm/chart-testing-action@v2.3.1
```

**Review by Skarlso — APPROVED** [review]
LGTM with a single question.:)

File: .github/workflows/helm.yml (hunk 1 above)
Comment (Skarlso) [reply .github/workflows/helm.yml:39]: not a blocker. :)

Comment (Skarlso) [reply .github/workflows/helm.yml:41]: Nice, thanks!

**Pull request merged**